### ISSUE TITLE
cloudflare:sockets error @ 0.4.8

### ISSUE THREAD

**Issue by mgarf** [neondatabase/serverless#20]
## Steps to reproduce
Setup Next.JS with serverless package and Drizzle ORM

## Expected result
No Error on `npm run dev`

## Actual result
```
cloudflare:sockets
Module build failed: UnhandledSchemeError: Reading from "cloudflare:sockets" is not handled by plugins (Unhandled scheme).
Webpack supports "data:" and "file:" URIs by default.
You may need an additional plugin to handle "cloudflare:" URIs.
```

## Logs, links
-

**Comment by mgarf** [comment]
Seems related to https://github.com/brianc/node-postgres/issues/2975

**Comment by jawj** [comment]
Ah: seems like we should hold back the `pg` version we rely on until we've investigated how Cloudflare sockets support works. I've just published 0.4.9 that does that — does this new version work for you?

**Comment by jawj** [comment]
@mgarf Could you confirm if 0.4.9 works, or are you still having problems?
